### PR TITLE
{devel}[dummy] ant, JUnit, Java-Hamcrest (REVIEW)

### PR DESCRIPTION
**Pull request by gjbex**
Note: depends on easybuild-easyblocks PR https://github.com/hpcugent/easybuild-easyblocks/pull/548
- ant
  - JUnit
  - Java-Hamcrest

### CODE REVIEW

**Comment by gjbex** [comment]
Test report by @gjbex
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.6
See https://gist.github.com/c5b303882bd74d9f2139 for a full test report.

**Comment by wpoely86** [comment]
ok to test

File: easybuild/easyconfigs/a/ant/ant-1.9.4-Java-1.8.0_31.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name = 'ant'
+version = '1.9.4'
+
+homepage = 'http://ant.apache.org/'
+description = """Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/ant/ant-1.9.4-Java-1.8.0_31.eb]: watch the 120 columns limit

File: easybuild/easyconfigs/j/JUnit/JUnit-4.12-Java-1.8.0_31.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+name = 'JUnit'
+version = '4.12'
+
+homepage = 'http://sourceforge.net/projects/junit'
+description = """A programmer-oriented testing framework for Java."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s.jar']
```
Comment (wpoely86) [inline easybuild/easyconfigs/j/JUnit/JUnit-4.12-Java-1.8.0_31.eb]: we don't have a `SOURCELOWER_JAR`? I should add that.

**Comment by boegel** [comment]
@gjbex: can you handle the tiny remark w.r.t. the long line in the ant easyconfig? if you're up for it, fix this in the other ant easyconfigs too

@wpoely86: care to submit a test report for this too?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 4 (3 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/7638cb446160a288f91d for a full test report.

**Comment by boegel** [comment]
uh-oh

**Comment by wpoely86** [comment]
The test script doesn't like it that the branch is called `feature/java_dev`

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 3 out of 4 (3 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/e6e35c7e01f0f533ab03 for a full test report.

**Comment by boegel** [comment]
@wpoely86: more/other problems?

**Comment by wpoely86** [comment]
No idea:
```
/home/ward/tmp/leeg/build/ant/1.9.4/dummy--Java-1.8.0_31/apache-ant-1.9.4/src/tests/junit/org/apache/tools/ant/taskdefs/ExecuteWatchdogTest.java:143: error: cannot access Matcher
                        throw new AssumptionViolatedException("process interrupted in thread", e);
                              ^
  class file for org.hamcrest.Matcher not found
```

**Comment by gjbex** [comment]
The problem you notice is because this PR relies on a fix of the ant easyblock as indicated in the original note (# 548 in easybuild-easyblock).  This is the missing Java Hamcrest stuff.

**Comment by boegel** [comment]
@gjbex: if you want to refer to a PR outside of the easybuild-easyconfigs repository, best copy-paste the URL to get a correct link: https://github.com/hpcugent/easybuild-easyblocks/pull/548

**Comment by migueldiascosta** [comment]
@gjbex, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**